gapic-generator/test/gapic/presenters/package/showcase_test.rb

```diff
@@ -54,9 +54,9 @@ def test_drift_showcase_v1beta1
     refute_nil echo_gprc
 
     assert_equal "::Google::Showcase::V1beta1::Echo::Client", echo_gprc[:libraryClient]
-    assert_equal 7, echo_gprc[:methods].length
+    assert_equal 7, echo_gprc[:rpcs].length
```

Comment:
Did you mean `echo_gprc` to be `echo grpc`?

Reply:
`echo_grpc` is a variable declared on the line 52 so unless I misunderstand the question, no, I meant `echo_grpc`.

Reply:
Sorry, my typo obscure the fact I was referring to your apparent typo: you say `gprc` instead of `grpc` (you have two letters transposed).